modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -3,27 +3,39 @@
 
 This section contains information about registering clients running {sle-micro} operating systems 5.1, 5.2, 5.3, 5.4, and 5.5 on {x86_64}, {arm64}, and {zsystems} (s390x) architectures.
 
-The {sle-micro} is an ultra-reliable, lightweight operating system purpose built for edge computing.
-It leverages the enterprise hardened security and compliance components of SUSE Linux Enterprise and merges them with a modern, immutable, developer-friendly OS platform.
-
 The {sle-micro} uses transactional updates.
-
 Transactional updates are atomic (all updates are applied only if all updates succeed) and support rollbacks.
 They do not affect a running system as no changes are activated until after the system is rebooted.
-This information is displayed in the menu:Systems[Details > Overview] subtab.
 
 For more information on transactional updates and rebooting, see
 https://documentation.suse.com/sles/html/SLES-all/cha-transactional-updates.html.
 
+// FIXME: We either must add this to the other client files (snippet), or
+//        up to the suse registration page
+The client registration described in this chapter, consists of these steps:
+
+* Synchronize software channels for the new client's operating system and architecture by using either {webui} or command line interface.
+
+ifeval::[{uyuni-content} == true]
+
+* Synchronize the client tool repository for the newly added operating systems by using command line interface.
+
+endif::[]
+
+* Run the actual registration.
```

Comment:
We have multiple ways to register the client, how about:

```suggestion
* Register the client with {productname} Server by using the {webui} or the command line interface.
```

Note: I see you use `command line interface` throughout the text, but I wonder if it should be a `command-line interface`. The way I previously used this, `command-line` is a compound adjective that modifies the word `interface` (which is why it requires a hyphen).

In my suggestion above, I use `command line interface` for consistency's sake.

Reply:
Thanks!  I think I now understood the thing with the dash!  Actually, it is also in Daria's wonderful https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-terminology . The Style Guide I must read more often!